### DISCUSSION TITLE
How can I import 2 folders of extracted features?

### DISCUSSION THREAD

**Discussion by phuvinhnguyen** [vitrivr/cineast#348] [Q&A]
I am having 2 different folders of extracted features from extracting my data. What should I do to import them to cottontaildb so that I can find the data from both folders? It seems that the later folder that I try to import will not be imported into cottontaildb.

**Comment by silvanheller** [comment]
Hey, unfortunately your question does not provide us with enough information to help with your issue. We provide documentation to import data here: https://github.com/vitrivr/cineast/wiki/Extraction-to-File-and-Import

If you run into errors, feel free to open an issue with the following information: Config files, steps to reproduce the issue and precise error logs.
